CHANGELOG/CHANGELOG-1.6.0.md

```diff
@@ -14,20 +15,31 @@
 - CSM Installation Wizard support for CSI PowerStore and PowerMax drivers and modules through Helm. ([#591](https://github.com/dell/csm/issues/591))
 - Support APIs for HostGroup in Powermax. ([#588](https://github.com/dell/csm/issues/588))
 - Add CSM Resiliency support for PowerStore. ([#587](https://github.com/dell/csm/issues/587))
-- Support PowerMax in CSM Observability. ([#586](https://github.com/dell/csm/issues/586))
 - Consistent Variable Name for RDM/vSphere. ([#584](https://github.com/dell/csm/issues/584))
 - CSM 1.6 release specific changes. ([#583](https://github.com/dell/csm/issues/583))
-- CSM support for Openshift 4.12. ([#571](https://github.com/dell/csm/issues/571))
 - Replication APIs to be moved from alpha phase. ([#432](https://github.com/dell/csm/issues/432))
+- PowerFlex preapproved GUIDs. ([#402](https://github.com/dell/csm/issues/402))
+- User friendly name to the PowerFlex. ([#181](https://github.com/dell/csm/issues/181))
+- Update Go version to 1.20 for CSM 1.6. ([#658](https://github.com/dell/csm/issues/658))
+- CSI Powermax- Snapshot restore to Metro Volumes. ([#609](https://github.com/dell/csm/issues/609))
+- Support PowerMax in CSM Observability. ([#586](https://github.com/dell/csm/issues/586))
 - PowerScale replication improvements. ([#573](https://github.com/dell/csm/issues/573))
 - PowerScale Replication: Implement Failback functionality. ([#558](https://github.com/dell/csm/issues/558))
 - PowerScale Replication - Reprotect should NOT be calling allow_write_revert. ([#532](https://github.com/dell/csm/issues/532))
 
 ### Bugs 
 
+- Replication : Creating extra snapshot for idempotent clone operation. ([#657](https://github.com/dell/csm/issues/657))
+- Update the optional parameters within angular brackets in upgrade operator page. ([#648](https://github.com/dell/csm/issues/648))
+- CSM Authorization CRD in the CSM Operator doesn't read custom configurations. ([#633](https://github.com/dell/csm/issues/633))
+- gobrick code owner file is containing errors. ([#568](https://github.com/dell/csm/issues/568))
+- PVC fails to resize with the message spec.capacity[storage]: Invalid value: "0": must be greater than zero. ([#507](https://github.com/dell/csm/issues/507))
+- Documentation - PowerStore Update Driver from v2.4 to v2.5 using Helm docs has wrong file name. ([#643](https://github.com/dell/csm/issues/643))
+- PowerMax Sample file is wrongly displaying SRDF group auto creation as false. ([#641](https://github.com/dell/csm/issues/641))
+- Observability - Improve Grafana dashboards for PowerFlex/Powerstore. ([#640](https://github.com/dell/csm/issues/640))
 - Powerscale csi driver while connecting smarconnect zone name first letter is getting truncated.. ([#617](https://github.com/dell/csm/issues/617))
+- "repctl cluster inject --use-sa" does not work when Replication is installed through `repctl`. ([#600](https://github.com/dell/csm/issues/600))
```

Comment:
This was fixed as [part of this PR. ](https://github.com/dell/csm-replication/pull/71)

Reply:
Thanks @ChristianAtDell , as long as the release milestone is accurate you should be all set. The changelog tracks all open and closed issues in the current release.